docs/resources/identity_acl.md

```diff
@@ -0,0 +1,70 @@
+---
+subcategory: "Identity and Access Management (IAM)"
+---
+
+# flexibleengine_identity_acl
+
+Manages a ACL resource within FlexibleEngine IAM service. The ACL allowing user access only from specified IP address
+ranges and IPv4 CIDR blocks. The ACL take effect for IAM users under the Domain account rather than the account itself.
+
+Note: You *must* have admin privileges in your FlexibleEngine cloud to use this resource.
+
+## Example Usage
+
+```hcl
+resource "flexibleengine_identity_acl" "acl" {
+  type = "console"
+
+  ip_cidrs {
+    cidr        = "159.138.39.192/32"
+    description = "This is a test ip address"
+  }
+  ip_ranges {
+    range       = "0.0.0.0-255.255.255.0"
+    description = "This is a test ip range"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `type` - (Required, String, ForceNew) Specifies the ACL is created through the Console or API. valid value are '
+  console' and 'api'. Changing this parameter will create a new ACL.
```

Comment:
only **console** is supported in Orange.

Reply:
fixed